src/extensions/default/PrefsCodeHints/main.js

```diff
@@ -38,11 +38,24 @@ define(function (require, exports, module) {
         JSONUtils           = brackets.getModule("language/JSONUtils"),
         Strings             = brackets.getModule("strings"),
         ThemeManager        = brackets.getModule("view/ThemeManager"),
+        CodeInspection      = brackets.getModule("language/CodeInspection"),
         _                   = brackets.getModule("thirdparty/lodash"),
-        data                = JSON.parse(require("text!data.json")),
+        languages           = LanguageManager.getLanguages(),
```

Comment:
You could do the same for the list of themes by listening to ExtensionManagers' statusChange event.